### PR TITLE
Conda tweaks

### PR DESCRIPTION
**Pull request by JoshKarpel**
Various tweaks to the text, some possibly-unnecessary infodumping and complication of the run script. You know, the standard Josh treatment.

### CODE REVIEW

File: conda-installation.md [hunk 1]
```diff
@@ -6,22 +6,35 @@ title: Create a Portable Python Installation with Miniconda
...
 
 When should you use Miniconda as an installation method in CHTC? 
-- Your software has specific Miniconda/conda centric installation instructions
-- The above is true *and* the software has a lot of dependencies
+- Your software has specific conda-centric installation instructions.
+- The above is true *and* the software has a lot of dependencies.
 - You mainly use Python to do your work. 
 
-There are two ways to use Miniconda on CHTC's HTC system. The first is to create your 
+Notes on terminology:
```
Comment (ChristinaLK) [inline conda-installation.md:19]: 😍

Comment (ChristinaLK) [reply conda-installation.md:19]: thanks for adding this.

File: conda-installation.md [hunk 2]
```diff
@@ -33,18 +46,20 @@ and then use a tool called `conda pack` to package it up for running jobs.
 
 ## 1. Create a Miniconda installation
 
-Download the latest Linux [miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it. 
+On the submit server, 
+download the latest Linux [miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it. 
 
 ```
+[alice@submit]$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
```
Comment (ChristinaLK) [inline conda-installation.md:53]: how stable is this link?

File: conda-installation.md [hunk 3]
```diff
@@ -137,29 +155,39 @@ When this step finishes, you should see a file in your current directory named
...
-of what is needed - change the `env-name` to the name of your environment. 
+of what is needed (customize as indicated to match your choices above).
 
 ```
 #!/bin/bash
 
+set -e
+
+# replace env-name on the right hand side of this line with the name of your conda environment
+env_name=env-name
```
Comment (ChristinaLK) [inline conda-installation.md:166]: I *almost* did this! Not everyone knows how bash variables work, so I'm on the fence about it. If we go the variable route, I would recommend:
```
ENVNAME=env-name
ENVDIR=$ENVNAME
```
at the start, and then proceed with the script.

File: conda-installation.md [hunk 4]
```diff
@@ -137,29 +155,39 @@ When this step finishes, you should see a file in your current directory named
...
 
 ## 5. Submit Jobs
 
 In your submit file, make sure to have the following: 
 
-- Your executable should be the the file you created in [step 4]()
-- Remember to transfer your primary script and the environment tar.gz file in 
-`transfer_input_files`. If the tar.gz file is larger than 100MB, please [email](mailto:chtc@cs.wisc.edu) us 
-about different tools for delivering the installation to your jobs. 
+- Your executable should be the the bash script you created in [step 4]().
```
Comment (ChristinaLK) [inline conda-installation.md:184]:
```suggestion
- Your executable should be the the bash script you created in [step 4](#4-create-a-job-executable).
```

File: conda-installation.md (hunk 2 above)
Comment (JoshKarpel) [reply conda-installation.md:53]: Very. I don't think it's changed in years.

File: conda-installation.md (hunk 3 above)
Comment (JoshKarpel) [reply conda-installation.md:166]: I'm torn... the programmer in me hates that the user has to change the name in multiple places (I, of course, missed one of the places the first time I tried the tutorial...), but doing it this way also makes this part ~mysterious~, when it's really pretty straightforward. I defer to your wisdom! Let me know which you prefer and I will make it so.

File: conda-installation.md (hunk 1 above)
Comment (JoshKarpel) [reply conda-installation.md:19]: :)

File: conda-installation.md (hunk 3 above)
Comment (ChristinaLK) [reply conda-installation.md:166]: The RCF team endorses variables. :)

**Comment by JoshKarpel** [comment]
Review comments resolved; @ChristinaLK  can you take another look?

**Pull request merged**